tests/unit/test_cli.py

```diff
@@ -4,6 +4,7 @@
 from unittest.mock import patch
 
 import pytest
+from django.core.management import CommandError, call_command
 from django.db.utils import OperationalError
```

Comment:
It's probably better like you did it (according to the [Django docs](https://docs.djangoproject.com/en/stable/ref/exceptions/#database-exceptions)):

```suggestion
from django.db import OperationalError
```